example/methods_directory/physical/psp.json

```diff
@@ -1,49 +1,52 @@
-    "path": "/qm/wf/psp/paw",
+    "slug": "pseudopotential",
     "categories": {
         "tier1": "qm",
         "tier2": "wf",
         "type": "psp",
         "subtype": "paw"
     },
     "data": {
-        "pseudo": [
+        "element": "Ru",
+        "hash": "3349af97451e04faae570056fb6a110b",
+        "filename": "ru_pbe_dojo-oncv_0.4.upf",
+        "path": "/export/share/pseudo/ru/gga/pbe/dojo-oncv/0.4/nc/ru_pbe_dojo-oncv_0.4.upf",
+        "valenceConfiguration": [
             {
-                "hash": "4e058592231fc4e091ac0c92d87797b0",
-                "apps": ["vasp"],
-                "element": "Si",
-                "source": "vasp",
-                "version": "5.2",
-                "path": "/export/share/pseudo/si/gga/pbe/vasp/5.2/paw/default/POTCAR",
-                "type": "paw",
-                "exchangeCorrelation": {
-                    "approximation": "gga",
-                    "functional": "pbe",
-                    "path": "/pb/qm/dft/ksdft/gga?functional=pbe"
-                },
-                "filename": "POTCAR",
-                "schemaVersion": "0.2.0",
-                "name": "pseudopotential",
-                "valenceConfiguration": [
-                    {
-                        "orbitalName": "3S",
-                        "orbitalIndex": 1,
-                        "principalNumber": 3,
-                        "angularMomentum": 0,
-                        "occupation": 2
-                    },
-                    {
-                        "orbitalName": "3P",
-                        "orbitalIndex": 2,
-                        "principalNumber": 3,
-                        "angularMomentum": 1,
-                        "occupation": 2
-                    }
-                ],
-                "inSet": [],
-                "tags": []
+                "orbitalName": "4S",
+                "orbitalIndex": 1,
+                "principalNumber": 4,
+                "angularMomentum": 0,
+                "occupation": 2.0
+            },
+            {
+                "orbitalName": "4P",
+                "orbitalIndex": 2,
+                "principalNumber": 4,
+                "angularMomentum": 1,
+                "occupation": 6.0
+            },
+            {
+                "orbitalName": "4D",
+                "orbitalIndex": 3,
+                "principalNumber": 4,
+                "angularMomentum": 2,
+                "occupation": 7.0
+            },
+            {
+                "orbitalName": "5S",
+                "orbitalIndex": 4,
+                "principalNumber": 5,
+                "angularMomentum": 0,
+                "occupation": 1.0
             }
-        ]
-    }
+        ],
+        "apps": ["espresso"],
+        "exchangeCorrelation": { "functional": "pbe", "approximation": "gga" },
+        "name": "pseudopotential",
+        "source": "dojo-oncv",
+        "type": "nc",
+        "version": "0.4"
+    },
+    "source": { "info": {}, "type": "exabyte" }
```

Comment:
This source doesn't make sense, there's already one on line 47

Reply:
Seems this is not used (`web-app/src/application/imports/properties/server/startup.js`), shall we get rid of it?